plugins/woocommerce/templates/emails/plain/customer-cancelled-order.php

```diff
@@ -0,0 +1,81 @@
+<?php
+/**
+ * Customer cancelled order email (plain text)
+ *
+ * This template can be overridden by copying it to yourtheme/woocommerce/emails/plain/customer-cancelled-order.php.
+ *
+ * HOWEVER, on occasion WooCommerce will need to update template files and you
+ * (the theme developer) will need to copy the new files to your theme to
+ * maintain compatibility. We try to do this as little as possible, but it does
+ * happen. When this occurs the version of the template file will be bumped and
+ * the readme will list any important changes.
+ *
+ * @see https://woocommerce.com/document/template-structure/
+ * @package WooCommerce\Templates\Emails\Plain
+ * @version 10.0.0
+ */
+
+use Automattic\WooCommerce\Utilities\FeaturesUtil;
+
+defined( 'ABSPATH' ) || exit;
+
+$email_improvements_enabled = FeaturesUtil::feature_is_enabled( 'email_improvements' );
+
+echo "=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=\n";
+echo esc_html( wp_strip_all_tags( $email_heading ) );
+echo "\n=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=\n\n";
+
+/* translators: %1$s: Order number */
+$text = __( 'We\'re sorry to let you know that your order #%1$s has been cancelled.', 'woocommerce' );
+if ( $email_improvements_enabled ) {
+	/* translators: %1$s: Order number */
+	$text = __( 'We\'re getting in touch to let you know that your order #%1$s has been cancelled.', 'woocommerce' );
+}
```

Comment:
This is not rendered well.
I got this for the plain text email option
```
Were getting in touch to let you know that your order #642 has been
cancelled.
```

Image
<img width="1028" alt="Screenshot 2025-05-16 at 11 44 28 AM" src="https://github.com/user-attachments/assets/0051aaee-b1a4-4b2a-a431-d63e9dde8bcf" />